### PR TITLE
Project Euler Problem 06

### PR DESCRIPTION
**Pull request by theja-vanka**

### CODE REVIEW

**Comment by Rishit-dagli** [comment]
@theja-vanka Could you sign the CLA mentioned in [this comment](https://github.com/Rishit-dagli/Python3-by-practice/pull/86#issuecomment-705546825) to move forward with this PR.

**Comment by theja-vanka** [comment]
Hey done. Let me know if I have anything else that I may have to comply with.

File: Project Euler/problem_06/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The sum of the squares of the first ten natural numbers is,
```
Comment (Rishit-dagli) [inline Project Euler/problem_06/README.md:1]: To maintain consistency could you add 

```
### Problem Statement :
```

at the start.

File: Project Euler/problem_06/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+12 + 22 + ... + 102 = 385
+The square of the sum of the first ten natural numbers is,
+
+(1 + 2 + ... + 10)2 = 552 = 3025
+Hence the difference between the sum of the squares of the first ten natural numbers and the square of the sum is 3025 − 385 = 2640.
+
+Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.
```
Comment (Rishit-dagli) [inline Project Euler/problem_06/README.md:9]: Are there any sample test cases, available if yes please add them

**Review by Rishit-dagli — CHANGES_REQUESTED** [review]

**Comment by theja-vanka** [comment]
Please Check now.

**Review by Rishit-dagli — APPROVED** [review]
This looks good to me, I will go ahead and merge it. Thank you for your contribution 💯 Feel free to support this repo by leaving a :star: !

**Comment by theja-vanka** [comment]
Thank you very much !! Enjoyed the privilage to contribute.

**Comment by Rishit-dagli** [comment]
@all-contributors please add @theja-vanka for code

**Pull request merged**